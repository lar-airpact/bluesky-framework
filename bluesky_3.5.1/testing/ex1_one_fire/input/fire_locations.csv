id,longitude,latitude,date_time,area,event_name
558,-119.278,37.2970009,20130730,14000,Aspen Fire