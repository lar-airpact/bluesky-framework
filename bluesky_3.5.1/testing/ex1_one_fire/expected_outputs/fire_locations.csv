id,event_id,latitude,longitude,type,area,date_time,slope,state,country,fips,scc,fuel_1hr,fuel_10hr,fuel_100hr,fuel_1khr,fuel_10khr,fuel_gt10khr,shrub,grass,rot,duff,litter,moisture_1hr,moisture_10hr,moisture_100hr,moisture_1khr,moisture_live,moisture_duff,consumption_flaming,consumption_smoldering,consumption_residual,consumption_duff,min_wind,max_wind,min_wind_aloft,max_wind_aloft,min_humid,max_humid,min_temp,max_temp,min_temp_hour,max_temp_hour,sunrise_hour,sunset_hour,snow_month,rain_days,heat,pm25,pm10,co,co2,ch4,nox,nh3,so2,voc,VEG,canopy,event_name,fccs_number
558,558,37.2970009,-119.278,Unknown,14000.0,201307300000-07:00,10.0,CA,USA,06019,2810090000,0.9,3.1,3.0,1.0,1.0,1.0,0.0,0.04,1.8,4.6,0.02,10.0,12.0,12.0,22.0,130.0,150.0,19.4415867986,20.3255788626,33.8834384666,50.222933005,6.0,6.0,6.0,6.0,40.0,80.0,13.0,30.0,4,14,6,18,5,8,3264425987747.127,14568.396793999995,17190.70821,178559.13228700001,1503286.2562630002,8507.727303000001,1345.84514,2909.4340220000004,1008.391539,41823.113989,Red fir forest,24.97,Aspen Fire,17
